controllers/v1/usersController.js

```diff
@@ -589,5 +591,104 @@ module.exports = class Users {
+     * @apiVersion 1.0.0
+     * @apiName List Observation Info
+     * @apiGroup users
+     * @apiHeader {String} X-authenticated-user-token Authenticity token
+     * @apiSampleRequest /assessment/api/v1/users/listObservationInfo
+     * @apiParamExample {json} Response:
+{
+    "message": "Observation overview information fetched successfully",
+    "status": 200,
+    "result": [
+        {
+            "_id": "60caf260ecea0772d81d7e5e",
+            "entities": [
+                "8ac1efe9-0415-4313-89ef-884e1c8eee34",
+                "b54a5c6d-98be-4313-af1c-33040b1703aa",
+                "2f76dcf5-e43b-4f71-a3f2-c8f19e1fce03"
+            ],
+            "isAPrivateProgram": false,
+            "name": "AP-TEST-PROGRAM-3.6.5-OBS-1-DEO",
+            "description": "Description of AP-TEST-PROGRAM-3.6.5-OBS-1-DEO",
+            "status": "published",
+            "submissionInfoArray": [
+                [
+                    {
+                        "_id": "60d1d97ea2709472c8c49b09",
+                        "entityId": "8ac1efe9-0415-4313-89ef-884e1c8eee34",
+                        "entityInformation": {
+                            "name": "SRIKAKULAM"
+                        },
+                        "entityType": "district",
+                        "createdBy": "fca2925f-1eee-4654-9177-fece3fd6afc9",
+                        "status": "started",
+                        "title": "Observation 1"
+                    }
+                ],
+                [
+                    {
+                        "_id": "60d58ee5983c1372591ccb5b",
+                        "entityId": "b54a5c6d-98be-4313-af1c-33040b1703aa",
+                        "entityInformation": {
+                            "name": "VIZIANAGARAM"
+                        },
+                        "entityType": "district",
+                        "createdBy": "fca2925f-1eee-4654-9177-fece3fd6afc9",
+                        "status": "completed",
+                        "title": "Observation 1"
+                    }
+                ],
+                [
+                    {
+                        "_id": "60caf262ecea0772d81d7e5f",
+                        "entityId": "2f76dcf5-e43b-4f71-a3f2-c8f19e1fce03",
+                        "entityInformation": {
+                            "name": "ANANTAPUR"
+                        },
+                        "entityType": "district",
+                        "createdBy": "fca2925f-1eee-4654-9177-fece3fd6afc9",
+                        "status": "started",
+                        "title": "Observation 1"
+                    }
+                ]
+            ]
+        }]}
+     * @apiUse successBody
+     * @apiUse errorBody
+     */
+     
+    /**
+    * list Observation Info
+    * @method
+    * @name listObservationInfo
+    * @param {Object} req -request Data.
+    * @returns {JSON} - List of observation consumed by an user
+    */
+    usersObservation(req) {
```

Comment:
can you move this controller in observationController fie @borkarsaish65

Reply:
Done